### DISCUSSION TITLE
Automate reliable formulae?

### DISCUSSION THREAD

**Discussion by codefromthecrypt** [Homebrew/discussions#2838] [Tap maintenance and Homebrew development] [answered]
I noticed this and was wondering if something could happen like this in homebrew. Basically it scans 1k packages and automates bumps so that projects don't need to raise bump PRs. https://github.com/vedantmgoyal2009/winget-pkgs-automation

I know many projects can be complex, but perhaps simple or reliable ones can become magically automated like this? wdyt?

**Comment by nandahkrishna** [comment, marked as answer]
Homebrew does this already for some formulae: see https://github.com/Homebrew/homebrew-core/blob/master/.github/workflows/autobump.yml.

**Reply by codefromthecrypt** [reply]
great. I'll remove the automation then
